### ISSUE TITLE
Waiting-video-stream feature

### ISSUE THREAD

**Issue by JacekLange** [WWBN/AVideo#9641]
Hello Daniel.

Could you implement in the streaming settings the ability to add a video file (preferably from an existing video database) that will play while waiting for a connection to the target obs client?

**Comment by DanielnetoDotCom** [comment]
Good idea, I will add it in my todo list, but there is no ETA for it, unless some one sponsor this feature

**Comment by JacekLange** [comment]
Maybe add it as a paid plugin? I suspect there will be a lot of willing customers.
